README.md

```diff
@@ -185,3 +163,23 @@ may request access to complete setup of configured shares.
     default:  * Debugger is active!
     default:  * Debugger PIN: 110-403-136
 ````
+
+### Containers
+
+The application can run with Docker.
+
+#### Development
+
+1. Create a docker image with name and tag targetted for `dev`: 
+   1. `docker build --target dev -t todo-app:dev .`
+2. Start the container with `docker-compose` 
+   1. `docker-compose up -d`
```

Comment:
Do we need to run this detached?

Reply:
My thinking was this separates the compose execution from the container but I guess it's not needed